src/Core/Install.php

```diff
@@ -0,0 +1,326 @@
+<?php
+/**
+ * @file src/Core/Install.php
+ */
+namespace Friendica\Core;
+
+use Friendica\BaseObject;
+use Friendica\Database\DBStructure;
+use Friendica\Object\Image;
+use Friendica\Util\Network;
+
+use Exception;
+use DOMDocument;
+
+
+/**
+ * @file include/Core/Install.php

```

Comment:
This docblock is duplicated above and below, is it still needed?

Reply:
I took it from src/Core/System.php and thought better make it the same way ;-)

But it isn't necessary, you're right